aikau/src/main/resources/alfresco/services/ActionService.js

```diff
@@ -1078,34 +970,6 @@ define(["dojo/_base/declare",
       },
 
       /**
-       * @instance
-       * @param {object} payload
-       */
-      onSyncLocation: function alfresco_services_ActionService__onSyncLocation(/*jshint unused:false*/ payload) {
-         var node = lang.clone(this._currentNode.parent);
-         var record = {
-            nodeRef: node.nodeRef,
-            displayName: node.properties["cm:name"],
-            jsNode: new JsNode(node)
-         };
-         this.callLegacyActionHandler("onActionCloudSync", record);
-      },
-
-      /**
-       * @instance
-       * @param {object} payload
-       */
-      onUnsyncLocation: function alfresco_services_ActionService__onUnsyncLocation(/*jshint unused:false*/ payload) {
-         var node = lang.clone(this._currentNode.parent);
-         var record = {
-            nodeRef: node.nodeRef,
-            displayName: node.properties["cm:name"],
-            jsNode: new JsNode(node)
-         };
-         this.callLegacyActionHandler("onActionCloudUnsync", record);
-      },
-
-      /**
```

Comment:
Yes, we're not using the legacy action code - they wouldn't have worked anyway. We have JIRA issues to address these actions properly in the future